### PR TITLE
Dop/test/accessibility fixes

### PR DESCRIPTION
**Pull request by sweco-seqtob**
Multiple fixes to increase the accessibility of the site

Includes :

Change language
Change of defaultcolors to get better contrast
Added label to search
Shortcuts for screenreaders in top of root-node
Fixes in documentHandler
Window title is now a H1
Buttons to panelheader now flex correctly

### CODE REVIEW

File: new-client/src/components/Search/SearchBar.js [hunk 1]
```diff
@@ -352,13 +352,16 @@ class SearchBar extends React.PureComponent {
     return (
       <TextField
         {...params}
-        label={undefined}
+        label={<Typography variant="srOnly">dsadsad</Typography>}
```
Comment (jacobwod) [inline new-client/src/components/Search/SearchBar.js:355]: Just wondering if you really want a `label` with that value on this TextField?

File: new-client/src/components/SrShortcuts/SrShortcuts.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from "react";
+import { List, ListItem, Link } from "@material-ui/core";
+
+/*
+ * @summary Small component to render a shortcut-menu high up in the DOM for screen readers
+ * @description The purpose of the comonent is to make the site more accessible by giving screen readers
```
Comment (jacobwod) [inline new-client/src/components/SrShortcuts/SrShortcuts.js:6]: comonent -> component

File: new-client/src/plugins/DocumentHandler/documentWindow/DocumentWindowBase.js [hunk 3]
```diff
@@ -177,6 +177,7 @@ class DocumentWindowBase extends React.PureComponent {
         this.bindSubscriptions();
 
         if (this.shouldShowDocumentOnStart()) {
+          console.log(this.props.options, "options");
```
Comment (jacobwod) [inline new-client/src/plugins/DocumentHandler/documentWindow/DocumentWindowBase.js:180]: Please remove

**Review by jacobwod — CHANGES_REQUESTED** [review]
Well done, just some minor fixes before we merge!

File: new-client/src/components/Search/SearchBar.js (hunk 1 above)
Comment (sweco-seqtob) [reply new-client/src/components/Search/SearchBar.js:355]: Wow, that was embarrasing

File: new-client/src/plugins/DocumentHandler/documentWindow/DocumentWindowBase.js (hunk 3 above)
Comment (sweco-seqtob) [reply new-client/src/plugins/DocumentHandler/documentWindow/DocumentWindowBase.js:180]: Thanks, missed one console log

**Review by jacobwod — APPROVED** [review]

**Comment by jacobwod** [comment]
@sweco-seqtob, I'm afraid we've just introduced some weird bugs here. 

Try overriding palette in `customTheme.json` to light:
```
{
  "palette": {
    "type": "light"
  }
}
```

User agents that previously used the perfectly nice dark mode will now be met with white text on white background.

Skärmavbild 2021-02-08 kl  12 57 59

Notice foreground color, e.g. in panel header.
Skärmavbild 2021-02-08 kl  12 58 05

**Comment by sweco-seqtob** [comment]
Ill add a ticket in out JIRA-board for this one

**Comment by jacobwod** [comment]
> Ill add a ticket in out JIRA-board for this one

I'm on it @sweco-seqtob, upcoming commit in `develop`!

**Pull request merged**